modules/client/src/main/java/org/apache/ignite/internal/client/ReliableChannel.java

```diff
@@ -568,7 +576,7 @@ private synchronized boolean initChannelHolders() {
 
             // Create new holders for new addrs.
             if (!curAddrs.containsKey(addr)) {
-                ClientChannelHolder hld = new ClientChannelHolder(new ClientChannelConfiguration(clientCfg, addr));
+                ClientChannelHolder hld = new ClientChannelHolder(new ClientChannelConfiguration(clientCfg, addr), requiredFeature);
```

Comment:
`ClientChannelHolder` is not static and should be able to access `requiredFeature`?

Reply:
Fixed, thanks